sgf-viewer/src/content.ts

```diff
@@ -0,0 +1,97 @@
+                ]
+            },
+            {
+                "title": "KataGo with 64 visits (level of top world professionals)",
+                "dir_name": "64_visits",
+                "server": "dqn.ist.berkeley.edu",
+                "_path_comment": "Computed by `grep -l v=1024 * | head -n 10` in /nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs, see https://www.notion.so/chaiberkeley/match-cp505-64-vs-adv-1_to_8192-39e0e303cd3f45199f4ce638c92556a5 for experiment log",
+                "paths": [
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/033FE05C5431ED13.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/048A6592BB515609.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/0FF3484FD5147146.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/14F7EEBACD86547C.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/16E93C54A1C3773E.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/187A346B3CC632E1.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/1CEF0F7800D4E196.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/1DF2F4ED5B859942.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/1EEA60F118BBB86C.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/2172599E22680D06.sgfs"
+                ],
+                "paths_1024": [
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/00FB73C086F202A1.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/00909067ADE53F5A.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/02B74D9148F158A4.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/02BFDE5F2ABA9C17.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/02EC7837398A88A5.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/0BDC4333B1246AA1.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/0EB3ADD4EAD8FE3F.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/1F2744213F4A1059.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/2132D23369A7A7FF.sgfs",
+                    "/nas/ucb/tony/go-attack/matches/cp505-v64-vs-adv-1-to-8192/sgfs/219DBF949E5229CB.sgfs"
+                ],
+                "max_games": 10,
+                "adversary": "34.1 million training steps, 8192 visits",
+                "victim": "Latest (cp505-v1-MCTS), 64 visits",
```

Comment:
should be v64 instead of v1? but also I don't think we ever explain this notation of cp505-v1 so maybe we should just remove it

Reply:
Agreed, seems confusing, removed 👍